### DISCUSSION TITLE
No Ads in Deezer with Maxthon

### DISCUSSION THREAD

**Discussion by Wolfoxfu** [uBlockOrigin/uAssets#23516] [General]
Salut, en attendant que le problèmes des pubs sur deezer sur les autres navigateurs soit réglé j'ai remarqué qu'utiliser ublock origin avec Maxthon fonctionne pour bloquer les pubs de deezer et la lecture reste fluide.

**Comment by krystian3w** [comment]
Maxthon 7.1.x?

I assume that they incorrectly detect whether the browser supports something and that's why there are no advertising inserts in the player.
